odbc/src/api/functions.rs

```diff
@@ -3557,7 +3547,8 @@ unsafe fn set_connect_attrw_helper(
                 conn.attributes.write().unwrap().login_timeout = Some(value_ptr as u32);
                 SqlReturn::SUCCESS
             }
-            ConnectionAttribute::SQL_ATTR_APP_WCHAR_TYPE => SqlReturn::SUCCESS,
+            ConnectionAttribute::SQL_ATTR_APP_WCHAR_TYPE
+            | ConnectionAttribute::SQL_ATTR_QUIET_MODE => SqlReturn::SUCCESS,
```

Comment:
uhm, I don't really know why someone would do that, but won't we return SQL_ERROR if an application calls SQLGetConnAttr(SQL_ATTR_QUIET_MODE) while we still let them set it to whatever?
To be safe, can we default to Null (quiet) and store the value passed-in here so that we return whatever was set/the default when SQLGetConnAttr is called.